### PR TITLE
Add PIDONE VA (ipv4)

### PR DESCRIPTION
**Pull request by lmiccini**
This commit add a "PIDONE Validated Architecture" composed by:

- 3 x ocp masters
- 3 x ocp workers
- 1 x testOperator-tainted ocp worker

The workers are required so that we can perform disruptive tests simulating ocp failures (reboots/crashes/etc).

Similarly, we need a special worker that would only run testoperator pods so that the tests will not be affected by the simulated infrastructure failures.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1752

### CODE REVIEW

File: va/pidone/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# VA: PIDONE
+
+If you are looking for information on how to deploy the PIDONE VA, then
+please see the
+[PIDONE README in the examples directory](../../../examples/va/PIDONE/README.md).
+
+This directory, `architecture/va/nfv/sriov/`, exists so that the
```
Comment (fultonj) [inline va/pidone/README.md:7]: Please change `architecture/va/nfv/sriov/` to `architecture/va/pidone/`

**Review by fultonj — COMMENTED** [review]
Please create `architecture/examples/va/pidone/README.md` and add basic instructions for someone to use to deploy the VA manually. For example, see `architecture/examples/va/hci/README.md`. Once this is created please patch the main README to link to this it. 
I did the following and the CRs look reasonable to me.
```
cd architecture/examples/va/pidone/
cd nncp
kustomize build . > nncp.yaml
cd ..
kustomize build . > control-plane.yaml
cd edpm/nodeset
kustomize build . > edpm.yaml
cd ../deployment
kustomize build . > deploy.yaml
```
This will need to be rebased (right now if I use it to build the HCI CRs they're missing the change from PR246).

**Comment by fultonj** [comment]
I don't have any objections to this. I think it can be merged after a few issues raised above are addressed.

File: automation/vars/pidone.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            kind: Node
+            resource_name: worker-3
+            state: patched
+        path: examples/va/pidone/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
```
Comment (abays) [inline automation/vars/pidone.yaml:26]: Would recommend maybe doubling or tripling this to avoid false failures due to slow NNCP processing

**Review by abays — APPROVED** [review]
/lgtm
/approve

**Comment by leifmadsen** [comment]
> I'm +2 to merge this.
> 
> @abays or @leifmadsen are you OK with that?

Sorry, I won't have time today to review this thoroughly, but I trust you guys, so I'm +1 to see it merge.

**Review by fultonj — APPROVED** [review]
/lgtm
/approve

**Comment by abays** [comment]
recheck

**Comment by abays** [comment]
recheck

**Pull request merged**